cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -255,6 +255,16 @@ rules:
   - network-attachment-definitions
   verbs:
   - get
+- apiGroups:
```

Comment:
This schema would also be needed in the helm chart and integration tests

Reply:
Could this also go in a `kind: Role` such as `rook-ceph-system`, instead of in the `ClusterRole`? It's a namespaced resource, so I would think it would work better there.